docs/architecture/decision-records/GithubActionWorkflow.md

```diff
@@ -0,0 +1,23 @@
+# GitHub Action (GHA) Reusable Workflows
+
+## Context
+
+The main GHA workflow file (continuous-deployment.yml) had grown in size and complexity. To improve readability and maintainability job steps will be grouped into reusable workflows. To use a GHA reusable workflow the yaml file **must** reside directly in the path `.github/workflows`. Therefore, subfolders cannot be used to organize files in this path.
+
+## Decision
+
+To alleviate confusion and improve on organizing GHA reusable workflows in the path `.github/workflows`, it was decided to add a prefix to the filename to provide context of the yaml file's purpose.
+
+- ***sub-***
```

Comment:
I think instead of the bold _and_ italic syntax in a bullet, I'd probably go with a level 3 heading. So something like:

> ### prefix
> Represents...